tests/resources/functions/node-long-coldstart/index.js

```diff
@@ -7,5 +7,5 @@ function fibo(n) {
 let cache = fibo(45);
 
 module.exports = async (context) => {
-    return context.res.send(cache);
+    return context.res.send(`${cache}`);
```

Comment:
This should now be fixed, you can remove it. Should work as before.

Reply:
Not working without it as of now. Would wait for re-release of OPR to test again.